scripts/o3de/o3de/android_support.py

```diff
@@ -170,6 +170,11 @@ def register_setting(key: str, description: str, default: str= None,  is_passwor
 SETTINGS_EXTRA_CMAKE_ARGS       = register_setting(key='extra.cmake.args',
                                                    description='Optional string to set additional cmake arguments during the native project generation within the android gradle build process')
 
+# CARBONATED -- begin : currently we support a sole install-time asset pack with size up to 1.5 Gb
+CARBONATED_AAB_ASSET_PACK_NAME  = register_setting(key='carbonated.aab.asset.pack.name',
```

Comment:
same here, let's omit all the carbonated qualifiers

Reply:
Removed